ee/benchmarks/benchmarks.py

```diff
@@ -0,0 +1,115 @@
+# isort: skip_file
+# Needs to be first to set up django environment
+from .helpers import *
+
+from datetime import timedelta
+from typing import List, Tuple
+
+from ee.clickhouse.client import sync_execute
+from ee.clickhouse.materialized_columns import backfill_materialized_columns, get_materialized_columns, materialize
+from ee.clickhouse.queries.trends.clickhouse_trends import ClickhouseTrends
+from posthog.models import Team, Organization
+from posthog.models.filters.filter import Filter
+from posthog.models.property import PropertyName, TableWithProperties
+
+MATERIALIZED_PROPERTIES: List[Tuple[TableWithProperties, PropertyName]] = [("events", "$host"), ("person", "email")]
+
+DATE_RANGE = {"date_from": "2021-01-01", "date_to": "2021-10-01"}
+SHORT_DATE_RANGE = {"date_from": "2021-07-01", "date_to": "2021-10-01"}
+
+
+class QuerySuite:
+    timeout = 3000.0  # Timeout for the whole suite
+
+    team: Team
+
+    def setup(self):
+        for table, property in MATERIALIZED_PROPERTIES:
+            if property not in get_materialized_columns(table):
+                materialize(table, property)
+                backfill_materialized_columns(table, [property], backfill_period=timedelta(days=1_000))
+
+        # :TRICKY: Data in benchmark servers has ID=2
```

Comment:
Q: Do we have a backup of the benchmark server? Or a deterministic way to recreate the same data?

I fear benchmarking becomes useless if we lose that server.

Reply:
> Or a deterministic way to recreate the same data?

Yep, we have this though it's a undocumented hack. Note the data is not randomly generated, don't want to put what it is in a public channel. :)

We can also backfill the benchmarks from commits if e.g. the hardware changes as well (I did this earlier).